evm/db/backends/level.py

```diff
@@ -0,0 +1,64 @@
+import shutil
+from .base import (
+    BaseDB,
+)
+
+
+class LevelDB(BaseDB):
+    db = None
+    db_path = None
+
+    # Creates db as a class variable to avoid level db lock error
+    def __init__(self, db_path=None):
+        if not db_path:
+            raise TypeError("Please specifiy a valid path for your database.")
+        try:
+            import leveldb
+        except ImportError:
+            raise ImportError("LevelDB requires the leveldb \
+                               library which is not available for import.")
+        if not LevelDB.db:
+            self.db_path = db_path
+            LevelDB.db = leveldb.LevelDB(db_path,
```

Comment:
This will mutate the `LevelDB` class for all future instantiations of the `LevelDB` class which is *bad* as it will bleed state between different db instances.  This should most definitely be using `self.db`.